sql/V6_MoreUserFields.sql

```diff
@@ -0,0 +1,11 @@
+
+ALTER TABLE users 
+    ADD COLUMN email VARCHAR NOT NULL;
+    ADD COLUMN bio VARCHAR;
+    ADD COLUMN address1 VARCHAR;
+    ADD COLUMN address2 VARCHAR;
+    ADD COLUMN city VARCHAR;
+    ADD COLUMN state VARCHAR;
+    ADD COLUMN zip VARCHAR;
+    ADD COLUMN weight BIGINT NOT NULL;
```

Comment:
BIGINT supports values up to 9,223,372,036,854,775,807.
SMALLINT goes to 32,767, which is probably big enough. :)